src/transformers/models/bart/modeling_bart.py

```diff
@@ -950,38 +952,48 @@ def set_input_embeddings(self, value):
         self.encoder.embed_tokens = self.shared
         self.decoder.embed_tokens = self.shared
 
-    def get_output_embeddings(self):
-        return _make_linear_from_emb(self.shared)  # make it on the fly
```

Comment:
no need to make anything on the fly -> that's why we have the weight tying mechanism